hie.yaml

```diff
@@ -0,0 +1,25 @@
+cradle:
```

Comment:
this file has little to do with nix, I'd suggest a seperate PR.

Reply:
Eh, it's fine for this time, it's only one more file :)